### DISCUSSION TITLE
Domain transfer problems

### DISCUSSION THREAD

**Discussion by lKaniel** [vercel/community#6851] [Help] [answered]
### Summary

I want to transfer my domain to AWS from Vercel. I made a request, it succeeded and I even created a hosted zone in Route 53. But domain still exists in Vercel and points to Vercel's name servers. When I try to change it to AWS's ns, it said that it can't find it on upstream provider. Am I missing something, can you help me?

### Steps to Reproduce

Try to transfer a domain from Vercel to AWS

**Comment by amyegan** [comment]
Hey @lKaniel. If the transfer succeeded, you should see the domain on AWS. You would need to manage the domain from there instead of from your Vercel dashboard.

https://vercel.com/docs/projects/domains/troubleshooting

**Reply by lKaniel** [reply]
Yes, that's exactly what I did, I've created a hosted zone in AWS and have there name servers set to AWS, but domain still points to Vercel,  and I can even still manage it on Vercel platform. The thing is that my AWS configuration just does not apply to it, seems like Vercel's old one still works

**Reply by amyegan** [reply]
If you let me know the domain, I can check on more details for you

**Reply by lKaniel** [reply]
Yes, the domain is:

cryptob.io

Thank you

**Reply by amyegan** [reply, marked as answer]
I checked our system and confirmed Vercel is no longer the registrar. A Whois check shows that Amazon is currently the registrar: [whois.com/whois/cryptob.io](https://www.whois.com/whois/cryptob.io). 

Since the domain is still using Vercel nameservers, you can manage DNS records from your Vercel dashboard. Or you can **go to AWS to change the nameservers** if you prefer to manage the domain there.

If you need help changing the nameservers, please [reach out to the AWS team](https://aws.amazon.com/contact-us/). They'll be able to walk you through the steps and investigate any issues you experience. 

https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-name-servers-glue-records.html

**Reply by lKaniel** [reply]
Thank you for answers and your time, I was able to resolve a problem
